R/paths.R

```diff
@@ -11,12 +12,16 @@
 #'   `NULL`, in which case it will check the input DAG for exposure.
 #' @param adjust_for character vector, a set of variables to control for.
 #'   Default is `NULL`.
+#' @param limit numeric value. The maximum number of paths to consider when
+#'   identifying open paths. If `limit` is smaller than the number of paths
+#'   between `from` and `to`, potential open pathways may not be returned.
+#'   However, if `limit` is set too large, you may run into memory issues.
```

Comment:
Let's lean on dagitty for this

```suggestion
#' @inheritParams dagitty::paths
```

Reply:
I changed this as requested. 

Initially, I even did this, but then thought that the dagitty documentation does not make clear that it is not only that more than `limit` paths are not shown, but also that there is a fixed sequence in which paths are listed and for this reason even not all open paths are considered. I think that some (including me at the beginning) might think that if, say  `limit = 100` and you get back 6 open paths then there are no more than 6 open paths, which is not generally the case.

But I agree that it would be more effective to solve this from the dagitty side.

Reply:
Yes, I suggest you follow your instinct and offer a PR there with improvements